e2e/cypress/integration/ui/common/global_filter.spec.ts

```diff
@@ -58,24 +64,6 @@ describe('global projects filter', () => {
   });
 });
 
-function cleanupProjects(id_token: string): void {
```

Comment:
this isn't needed since we have our handy `cy.cleanupIAMObjectsByIDPrefixes` (which also knows how to delete a project's rules before deleting it)